.travis.yml

```diff
@@ -1,61 +1,10 @@
 language: cpp
 matrix:
   include:
-    # linux
-    - name: "ARCH=arm"
-      env: ARCH=arm
-    - name: "ARCH=arm64 LD=ld.lld"
-      env: ARCH=arm64 LD=ld.lld-8
-    - name: "ARCH=ppc64le"
-      env: ARCH=ppc64le
-    - name: "ARCH=x86_64"
-      env: ARCH=x86_64
-    # linux (cron only)
-    - name: "ARCH=arm64"
-      env: ARCH=arm64
-      if: type = cron
-    # linux-next (cron only)
-    - name: "ARCH=arm REPO=linux-next"
-      env: ARCH=arm REPO=linux-next
-      if: type = cron
-    - name: "ARCH=arm64 REPO=linux-next"
-      env: ARCH=arm64 REPO=linux-next
-      if: type = cron
-    - name: "ARCH=arm64 LD=ld.lld REPO=linux-next"
-      env: ARCH=arm64 LD=ld.lld-8 REPO=linux-next
-      if: type = cron
-    - name: "ARCH=ppc64le  REPO=linux-next"
-      env: ARCH=ppc64le REPO=linux-next
-      if: type = cron
-    - name: "ARCH=x86_64 REPO=linux-next"
-      env: ARCH=x86_64 REPO=linux-next
-      if: type = cron
-    # stable
-    # TODO: re-enable once green
-    - name: "ARCH=arm64 REPO=4.19"
-      env: ARCH=arm64 REPO=4.19
-      if: type = cron
-    - name: "ARCH=x86_64 REPO=4.19"
-      env: ARCH=x86_64 REPO=4.19
-      if: type = cron
-    - name: "ARCH=arm64 REPO=4.14"
-      env: ARCH=arm64 REPO=4.14
-      if: type = cron
-    - name: "ARCH=x86_64 REPO=4.14"
-      env: ARCH=x86_64 REPO=4.14
-      if: type = cron
-    #- name: "ARCH=arm64 REPO=4.9"
-      #env: ARCH=arm64 REPO=4.9
-      #if: type = cron
-    #- name: "ARCH=x86_64 REPO=4.9"
-      #env: ARCH=x86_64 REPO=4.9
-      #if: type = cron
-    #- name: "ARCH=arm64 REPO=4.4"
-      #env: ARCH=arm64 REPO=4.4
-      #if: type = cron
-    #- name: "ARCH=x86_64 REPO=4.4"
-      #env: ARCH=x86_64 REPO=4.4
-      #if: type = cron
+    - name: "ARCH=arm64 REPO=4.4"
+      env: ARCH=arm64 REPO=4.4
+    - name: "ARCH=x86_64 REPO=4.4"
+      env: ARCH=x86_64 REPO=4.4
```

Comment:
assuming you're done testing as I'm marked for review (assign this PR to me when you're done to signal you're ready for review), we probably don't want to commit the deletions, and only want this for cron.

Reply:
Thanks, done.